FAO Publications.js

```diff
@@ -535,8 +607,8 @@ var testCases = [
 				"numPages": "251",
 				"place": "Rome, Italy",
 				"publisher": "FAO",
-				"series": "Non-wood forest products working paper",
-				"seriesNumber": "21",
+				"series": "Produits Forestiers Non-Ligneux",
+				"seriesNumber": "No. 21",
```

Comment:
Ack, this is a regression (probably because you're scraping the French site and the tests were originally created on the English one, so nobody caught it). Can we strip the "No." prefix? Maybe just match to find the numeric part so that it works across languages, like

```js
newItem.seriesNumber = (newItem.seriesNumber.match(/\d+/) || [])[0];
```

That will pull out the number if there is one, or `undefined` if there isn't.